mkdocs/docs/configuration.md

```diff
@@ -368,11 +362,84 @@ catalog:
     header.content-type: application/vnd.api+json
 ```
 
-Specific headers defined by the RESTCatalog spec include:
 
-| Key                                  | Options                               | Default              | Description                                                                                                                                                                                        |
-| ------------------------------------ | ------------------------------------- | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `header.X-Iceberg-Access-Delegation` | `{vended-credentials,remote-signing}` | `vended-credentials` | Signal to the server that the client supports delegated access via a comma-separated list of access mechanisms. The server may choose to supply access via any or none of the requested mechanisms |
+#### Authentication Options
+- **SigV4**: For AWS services that require SigV4 signing.
+- **OAuth2**: For services that require OAuth2 authentication.
```

Comment:
lets add these as 2 separate subsections. One for OAuth2, followed by SigV4. 
OAuth2 has options are `oauth2-server-uri`, `token`, `credential`, along with the few from my comment on top

SigV4 is an AWS specific protocol. these are all related to sigv4:`rest.sigv4-enabled`, `rest.signing-region`, `rest.signing-name`